doc/manual/source/interaction/geoserver_web-api.rst

```diff
@@ -1,14 +1,14 @@
 .. _geoserver_web-api:
 
 GeoServer REST configuration integration
-========================================
+########################################
 
 GeoGig integrates well with the standard GeoServer REST configuration API in order to configure vector datastores and layers from GeoGig repositories.
 
 You can use a combination of GeoGig's own "Web API" and GeoServer's REST configuration API to create a wide variety of scripts.
 
 GeoGig's plugin Web-API
```

Comment:
Please update all references of `web-API` to `web API` to match the rest of the documentation.